src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -112,11 +115,28 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
             try
             {
                 invitation.MeetingId = await CreateOutlookMeeting(request, meetingParticipants, invitation, project.Name);
+            }
+            catch
```

Comment:
I think we just should catch exceptions of type IpoSendMailException here. If CreateOutlookMeeting fails for other reason, don't retry with smtp